chat-plugins/datasearch.js

```diff
@@ -656,7 +656,7 @@ function runDexsearch(target, cmd, canAll, message) {
 	searches.sort((a, b) => Object.values(a).reduce(accumulateKeyCount, 0) - Object.values(b).reduce(accumulateKeyCount, 0));
 
 	let lsetData = {};
-	for (const group of searches) {
+	for (const group in searches) {
```

Comment:
Can't this just be ``(const alts of searches)``, since ``alts`` is only just the entry

Reply:
This should actually be `for (const alts of searches) {`